docs/installation/debian.md

```diff
@@ -12,113 +13,122 @@ parent = "smn_linux"
 
-If you need to add an HTTP Proxy, set a different directory or partition for the
-Docker runtime files, or make other customizations, read our Systemd article to
-learn how to [customize your Systemd Docker daemon options](/articles/systemd/).
+ 3. Purge any older repositories.
 
-> **Note**:
-> If you want to enable memory and swap accounting see
-> [this](/installation/ubuntulinux/#adjust-memory-and-swap-accounting).
+         $ apt-get purge lxc-docker*
+         $ apt-get purge docker.io*
 
-### Uninstallation
+ 4. Add the new `gpg` key.
 
-To uninstall the Docker package:
+         $ apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
 
-    $ sudo apt-get purge docker.io
+ 5. Open the `/etc/apt/sources.list.d/docker.list` file in your favorite editor.
 
-To uninstall the Docker package and dependencies that are no longer needed:
+     If the file doesn't exist, create it.
 
-    $ sudo apt-get autoremove --purge docker.io
+ 6. Remove any existing entries.
 
-The above commands will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
+ 7. Add an entry for your Debian operating system.
 
-    $ rm -rf /var/lib/docker
+     The possible entries are:
 
-You must delete the user created configuration files manually.
+         # Debian Wheezy
+        deb https://apt.dockerproject.org/repo debian-wheezy main
+        # Debian Jessie
+        deb https://apt.dockerproject.org/repo debian-jessie main
+        # Debian Stretch/Sid
+        deb https://apt.dockerproject.org/repo debian-stretch main
 
-## Debian Wheezy/Stable 7.x (64-bit)
+ 8. Save and close the file.
 
-Docker requires Kernel 3.8+, while Wheezy ships with Kernel 3.2 (for more details
-on why 3.8 is required, see discussion on
-[bug #407](https://github.com/docker/docker/issues/407)).
+ 9. Update the `apt` package index.
 
-Fortunately, wheezy-backports currently has [Kernel 3.16
-](https://packages.debian.org/search?suite=wheezy-backports&section=all&arch=any&searchon=names&keywords=linux-image-amd64),
-which is officially supported by Docker.
+         $ apt-get update
 
-### Installation
+ 10. Verify that `apt` is pulling from the right repository.
 
-1. Install Kernel from wheezy-backports
+         $ apt-cache policy docker-engine
 
-    Add the following line to your `/etc/apt/sources.list`
+     From now on when you run `apt-get upgrade`, `apt` pulls from the new apt repository.  
 
-    `deb http://http.debian.net/debian wheezy-backports main`
+### Prerequisites on Debian Jessie 8.0 (64-bit)
 
-    then install the `linux-image-amd64` package (note the use of
-    `-t wheezy-backports`)
+Debian 8 comes with a 3.16.0 Linux kernel. On Debian the Docker package and executable are called `docker.io` as the a much older KDE3/GNOME2 package already used the name `docker`. The `docker.io` package can be found
```

Comment:
The section I have VM with Debian stretch and Docker .  The binary is called `docker`.   


![image](https://cloud.githubusercontent.com/assets/1347209/10269335/4e030420-6a89-11e5-8ae8-bf5695343817.png)